angular/core/components/thread_page/activity_card/discussion_moved.haml

```diff
@@ -0,0 +1,10 @@
+.thread-item{ng-controller: "DiscussionMovedItemController"}
+  .thread-item__header.media
```

Comment:
Hrrmm.. are there examples of doing this without using .media? I think we want to avoid using .media if posssible.. but if there are no other examples then that's ok.